src/Wallabag/CoreBundle/Resources/views/Entry/entries.html.twig

```diff
@@ -21,11 +21,12 @@
     {% if entries is empty %}
         <div class="messages warning"><p>{% trans %}No articles found.{% endtrans %}</p></div>
     {% else %}
+        <div><form>{{ form_rest(form) }}<button class="btn waves-effect waves-light" type="submit" id="submit-filter" value="filter">Filter</button></form></div>
         {% for entry in entries %}
             <div id="entry-{{ entry.id|e }}" class="entrie">
```

Comment:
`entrie`, really? It's not english at all, it should be entry.
Maybe it should be done in a different PR, but you are using this class in your test, some maybe you can fix it